circleci/fabfile.py

```diff
@@ -237,11 +240,17 @@ def _run_remote_cwf_integ_test(repo: str, magma_root: str):
                 '-f docker-compose.integ-test.yml '
                 'build --parallel')
         test_xml = "tests.xml"
-        result = run('fab integ_test:'
-                     'destroy_vm=True,'
-                     'transfer_images=True,'
-                     'test_result_xml=' + test_xml,
-                     timeout=110*60, warn_only=True)
+        with settings(abort_exception=FabricException):
```

Comment:
what does this context manager do ?

Reply:
Right now if there is an exception we just abort the script. 

This allows us to define an exception object that can be caught instead of aborting while running integ_test. 

The idea is to make sure `_transfer_all_artifacts` is run and we can see the logs.